ui/js/react/treeherder/revision-list.jsx

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
Do we need to nest the react files in `ui/js/react/treeherder/revision-list.jsx` for build reasons?
If not, I'm wondering if we want to structure by function rather than react vs not? (Though I'll defer to Cameron on this)